### DISCUSSION TITLE
Saleor x Next.js (work-in-progress)

### DISCUSSION THREAD

**Discussion by karolkielecki** [saleor/saleor#7423] [Announcements]
We’re working on a new integration: Saleor + Next.js Commerce by Vercel, and we couldn't be more excited! Check out the working demo and let us know your thoughts.

Demo: https://saleor-commerce.vercel.app

---
- [Twitter](https://twitter.com/getsaleor/status/1399698268733464576)
- [LinkedIn](https://www.linkedin.com/feed/update/urn:li:activity:6805466921367031808)

**Comment by heliumbrain** [comment]
That looks really sweet! Excited to see the final result. The size-selector for some of the products looks a bit wonky, but I'm sure you are aware and it's a easy fix anyway.

Any idea on when this might go live?

![image](https://user-images.githubusercontent.com/1607668/120620852-abe75800-c45d-11eb-9143-8d9c86d22ef5.png)

**Reply by karolkielecki** [reply]
@heliumbrain Thanks for your kind words! I've reported this bug to the team--thanks! It should be available pretty soon, but I don't know the exact date. I'll keep you updated.

Cheers,
